Executable/raw_injector.py

```diff
@@ -0,0 +1,128 @@
+import sys
+import ctypes
+import ctypes.wintypes
```

Comment:
I think `from ctypes import wintypes` is better: no many imports + all types are not required to scroll to the beginning to find where it is from.